applications/teletype/src/teletype_util.erl

```diff
@@ -376,17 +388,34 @@ find_account_params(AccountId) ->
                                     | maybe_add_parent_params(AccountId, AccountJObj)
                                    ]);
         {'error', _E} ->
-            lager:debug("failed to find account doc for ~s: ~p", [AccountId, _E]),
+            ?LOG_DEBUG("failed to find account doc for ~s: ~p", [AccountId, _E]),
             []
     end.
 
+-ifdef(TEST).
+fetch_account_for_params(?AN_ACCOUNT_ID) -> {ok, fixture("an_account.json")};
+fetch_account_for_params(?A_MASTER_ACCOUNT_ID) -> {ok, fixture("a_master_account.json")};
+fetch_account_for_params(?MATCH_ACCOUNT_RAW(_)) -> {error, testing_too_hard}.
+-else.
+fetch_account_for_params(AccountId) -> kz_account:fetch(AccountId).
+-endif.
+
+-ifdef(TEST).
+-spec fixture(nonempty_string()) -> kz_json:object().
+fixture(JSONFileName) ->
```

Comment:
`read_preview_doc/1` reads things in priv/preview_data. This `fixture` function is a test-mode only function that fetches files from test/. Why should I use read_preview_doc?